hack/heap/hh_shared.c

```diff
@@ -592,7 +594,7 @@ void hh_add_dep(value ocaml_dep) {
   htable_add(deptbl, dep >> 31, dep);
```

Comment:
@mroch Much appreciated, thanks! (Also found FB infer recently, awesome work guys! love all what you do)